### DISCUSSION TITLE
Documentation request: clean does not touch .git, .github, .ssh?

### DISCUSSION THREAD

**Discussion by sanchom** [JamesIves/github-pages-deploy-action#697] [Ideas]
If I'm reading this right

https://github.com/JamesIves/github-pages-deploy-action/blob/7e40b83dc23ffc53e223142b565a44ef4a793313/src/git.ts#L137

the `clean` operation does not touch `.git`, `.github`, `.ssh` directories. Is that correct?

I had accidentally put a `.github` directory in my deployment destination. When I removed it from my folder to be published, I expected `clean` to get rid of it in the destination, but it didn't disappear! I had to remove it manually. I think this is intentional and probably the right behaviour.

Think it's worth mentioning this in the documentation for the `clean` option?

I made quick pull request: https://github.com/JamesIves/github-pages-deploy-action/pull/698

**Comment by JamesIves** [comment]
Good callout! I'll merge this later today. Thanks!
